aptos/token_bridge/sources/transfer_tokens.move

```diff
@@ -62,6 +68,34 @@ module token_bridge::transfer_tokens {
         )
     }
 
+    public entry fun transfer_tokens_with_payload_with_signer<CoinType>(
+        sender: &signer,
+        amount: u64,
+        recipient_chain: u64,
+        recipient: vector<u8>,
+        wormhole_fee: u64,
+        nonce: u64,
+        payload: vector<u8>
+        ): u64 acquires State {
+        let coins = coin::withdraw<CoinType>(sender, amount);
+        let wormhole_fee_coins = coin::withdraw<AptosCoin>(sender, wormhole_fee);
+
+        if (!exists<State>(signer::address_of(sender))) {
+            let emitter_cap = register_emitter();
+            move_to<State>(sender, State { emitter_cap: emitter_cap });
+        };
```

Comment:
I really wish there was another way to do these emitters... this emitter is going to be 110% meaningless to anybody else on the planet.  More importantly, it is going to be difficult to know exactly what your emitter address is until after you send your first message.  Maybe we need a specific API to potentially assign a emitter and return its value?